Revert upstream patch that enabled new clocksources.

Signed-off-by: Hannu Nyman <hannu.nyman@iki.fi>

 Upstream patch to be reverted:

 Subject: ARM: mvebu: enable ARM_GLOBAL_TIMER compilation Armada 38x platforms

 Armada 38x SoCs along with legacy timer (time-armada-370-xp.c),
 comprise generic Cortex-A9 global timer (arm_global_timer.c).
 Enable its compilation. The system clocksource subsystem
 will pick one of above two available ones in case the global
 timer node is present in the device tree.

--- a/arch/arm/mach-mvebu/Kconfig
+++ b/arch/arm/mach-mvebu/Kconfig
@@ -60,8 +60,6 @@ config MACH_ARMADA_38X
 	select ARM_ERRATA_720789
 	select PL310_ERRATA_753970
 	select ARM_GIC
-	select ARM_GLOBAL_TIMER
-	select CLKSRC_ARM_GLOBAL_TIMER_SCHED_CLOCK
 	select ARMADA_370_XP_IRQ
 	select ARMADA_38X_CLK
 	select HAVE_ARM_SCU
